ui/lib/core/addon/components/replication-summary-card.js

```diff
@@ -0,0 +1,52 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import layout from '../templates/components/replication-summary-card';
+
+/**
+ * @module ReplicationSummaryCard
+ * ReplicationSummaryCard components
+ *
+ * @example
+ * ```js
+ * <ReplicationSummaryCard
+    @title='States'
+    @replicationDetails=replicationDetails
+    />
+ * ```
+ * @param {string} [title=null] - The title to be displayed on the top left corner of the card.
+ * @param replicationDetails=null{DS.Model.replicationDetails} - An Ember data object off the Ember data model.  It is computed at the parent component and passed through to this component.
+ */
+
+export default Component.extend({
+  layout,
+  title: null,
+  replicationDetails: null,
+  lastDrWAL: computed('replicationDetails.dr.{lastWAL}', function() {
```

Comment:
i haven't tested this syntax, but do you know if this might need to be recomputed like `replicationDetails.{dr}.{lastWAL}` to ensure that the property is updated when dr changes too?

Reply:
I looked at the documentation, and suspect, though can't confirm that the double {} is not valid.  You can watch multiple properties, like this `replication.dr.{lastWAL, clusterId}`, but both the dr object, and the lastWAL property.  

However, lastWAL changes frequently, and that's the property we care about for the computed `lastDrWAL`.  Additionally, this is the same as watching as we do in the other dashboards, `replicationDetails.{lastWAL}`, it's just our object has another nested object inside of it.  The lastWAL will change whenever we enable, disabled, or make any cluster changes (adding a policy, token, etc.).

I'll do some testing today to make sure these computed properties change as we expect them to.